docs/modules/ROOT/pages/departments_version_2.adoc

```diff
@@ -1,6 +1,6 @@
-= Departments
+= Departments in Version 2.x
 
-RuboCop RSpec currently has four departments:
+RuboCop RSpec Version 2.x has four departments:
```

Comment:
We have the upgrade document in docs/modules/ROOT/pages/upgrade_to_version_3.adoc, so I don’t think we need to document which departments were available in version 2. Let’s just document the departments in ~v3~ master branch.

Reply:
I updated this PR. Thank you!